### PR TITLE
set incentive admin in instantiate

### PR DESCRIPTION
**Pull request by LaurensKubat**
# 1. Overview

Sets the incentive admin to a seperate address in the instantiate message instead of setting this to the instantiator address

## 2. Implementation details

Adds a field to the instantiate message and sets the incentive admin in state to that address
## 3. How to test/use

Updated test tube coverage should cover this

## 4. Checklist

- [x] Does the Readme need to be updated?

### CODE REVIEW

**Review by arhamchordia — DISMISSED** [review]
Looks good to me.

**Review by arhamchordia — DISMISSED** [review]
Looks good. CI is failing on test-tube (pull request).

File: smart-contracts/contracts/merkle-incentives/.cargo/config [hunk 1]
```diff
@@ -2,5 +2,5 @@
 wasm = "build --release --lib --target wasm32-unknown-unknown"
 unit-test = "test --lib"
 schema = "run --bin schema"
-test-tube = "test --package merkle-incentives --lib -- --include-ignored test_tube:: --nocapture --test-threads=1"
+test-tube = "test --package merkle-incentives --test merkle -- --include-ignored --nocapture --test-threads=1"
```
Comment (magiodev) [inline smart-contracts/contracts/merkle-incentives/.cargo/config:5]: Im not sure about the `--test merkle`, is that correct?

File: smart-contracts/contracts/merkle-incentives/src/msg.rs [hunk 2]
```diff
@@ -7,7 +7,9 @@ use crate::{
 };
 
 #[cw_serde]
-pub struct InstantiateMsg {}
+pub struct InstantiateMsg {
+    pub incentive_admin: String,
```
Comment (magiodev) [inline smart-contracts/contracts/merkle-incentives/src/msg.rs:11]: Annoying comment, this should be `incentives_admin`, i think we always refer to it in the plural form.

File: smart-contracts/contracts/merkle-incentives/.cargo/config (hunk 1 above)
Comment (magiodev) [reply smart-contracts/contracts/merkle-incentives/.cargo/config:5]: oh nvm you moved the files to another folder

Comment (LaurensKubat) [reply smart-contracts/contracts/merkle-incentives/.cargo/config:5]: Yeah, the imports work a bit different if we have all integration tests such as test-tube tests in the tests folder, but the testing modules also work just a bit nicer

File: smart-contracts/contracts/merkle-incentives/src/msg.rs (hunk 2 above)
Comment (LaurensKubat) [reply smart-contracts/contracts/merkle-incentives/src/msg.rs:11]: sounds good, will change

**Review by magiodev — APPROVED** [review]
lgtm!

**Review by arhamchordia — APPROVED** [review]
Lgtm

**Pull request closed without merging**